docs/Function library.md

```diff
@@ -582,6 +582,16 @@ count(
 
 (Also see examples for json-file for host and port, sets of files and working directory).
 
+### parquet-file (Rumble specific)
+
+Parses one or more parquet files and returns a sequence of objects.
```

Comment:
Does the function currently handle multiple parquet file inputs? The implementation seemed like it reads a single file unless I'm missing something.

Reply:
@CanBerker it does work with several files, as for json-file. For this it suffices to use wildcards, etc. This is exactly the same syntax used by the Spark input functions.

Reply:
I have extended the doc to make sure this is clear.